### PR TITLE
MM-9637 Fix text not visible when typing on Android search box

### PR DESCRIPTION
**Pull request by enahum**
#### Summary
Android started to respect the lineHeight, so if the lineHeight was bigger than the actual height of the TextInput the text was drawn outside of the visible area.

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-9637

### CODE REVIEW

File: app/components/channel_drawer/channels_list/channels_list.js [hunk 1]
```diff
@@ -106,7 +106,11 @@ class ChannelsList extends React.PureComponent {
             backgroundColor: changeOpacity(theme.sidebarHeaderTextColor, 0.2),
             color: theme.sidebarHeaderTextColor,
             fontSize: 15,
-            lineHeight: 66
+            ...Platform.select({
+                android: {
+                    marginBottom: -5
+                }
+            })
         };
```
Comment (saturninoabril) [inline app/components/channel_drawer/channels_list/channels_list.js]: I wonder if `searchBarInput` should be part of returned object of `getStyleSheet`. Same case with other changes below.

Comment (enahum) [reply app/components/channel_drawer/channels_list/channels_list.js]: No it should not, if we set it as part of the style then we cannot retrieve properties from it

Comment (saturninoabril) [reply app/components/channel_drawer/channels_list/channels_list.js]: Got it, thanks!

Comment (hmhealey) [reply app/components/channel_drawer/channels_list/channels_list.js]: We could retrieve properties from it using `StyleSheet.flatten`, but I don't think we need it here

**Pull request merged**